openapi-generator/src/main/resources/templates/java-micronaut/common/model/pojo.mustache

```diff
@@ -131,6 +131,11 @@ public class {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{{#vendorE
     {{/vars}}
 
     {{#requiredPropertiesInConstructor}}
+        {{^micronaut_serde_jackson}}
+    public {{classname}}() {
```

Comment:
I think it would be better for constructor to be private or package-private. Otherwise the point of required properties is lost.

Reply:
fixed